scpd/lms/static/sass/partials/base/_variables.scss

```diff
@@ -1,9 +1,10 @@
-// Fonts
-$sans-serif: 'Source Sans Pro', Trebuchet, 'Helvetica Neue', Helvetica, $verdana, sans-serif;
-$f-sans-serif: $sans-serif;
-$serif: 'Crimson Text', Georgia, Cambria, 'Times New Roman', Times, serif;
-$f-serif: $serif;
-$header-sans-serif: $sans-serif;
-
-// Import defaults
+@import 'variable_overrides/palette.scss';
+
+@import 'variable_overrides/colors.scss';
+
+@import 'variable_overrides/fonts.scss';
+
+@import 'variable_overrides/images.scss';
```

Comment:
I'd much prefer to call the parent directory simply `variables`.
Including the `overrides` suffix is a bit misleading because while everything is a variable,
not everything is actually an override.

Reply:
I tried with "variables" it wouldn't let me...

I could do `variables_custom`?